src/KingSystem/Resource/resModelResourceDivide.h

```diff
@@ -3,10 +3,7 @@
 #include <heap/seadDisposer.h>
 #include <prim/seadSafeString.h>
 #include "KingSystem/Resource/resHandle.h"
-
-namespace al {
-class ByamlIter;
-}
+#include "KingSystem/Utils/Byaml/Byaml.h"
```

Comment:
Here, `al::ByamlIter` should still be forward declared for compile time reasons.

Please move the `#include "KingSystem/Utils/Byaml/Byaml.h"` to the .cpp (which is the only place where the definition of al::ByamlIter is required) and forward declare in this header.